dockerfiles/tasks.py

```diff
@@ -42,10 +42,15 @@ def down(c, volumes=False):
     'webpack': 'Start webpack development server (default: False)',
     'ext-theme': 'Enable new theme from ext-theme (default: False)',
     'scale-build': 'Add additional build instances (default: 1)',
-    'ngrok': 'ngrok domain to serve the application. Example: "17b5-139-47-118-243.ngrok.io"',
+    'http-domain': 'Configure a production domain for HTTP traffic. Subdomains included, '
+                   'example.dev implies *.examples.dev for proxito. Example: '
+                   '"17b5-139-47-118-243.ngrok.io"',
+    'https': 'Forces HTTPS settings that are otherwise undetected when a :443 HTTPS proxy is in '
```

Comment:
We talked about removing the `--https` argument for now since it's not required for development.